schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py

```diff
@@ -406,6 +432,8 @@ class ExtendedValidationDepositionKeyPhotoSource(DepositionKeyPhotoSource):
 # ==============================================================================
 @alru_cache
 async def lookup_doi(doi: str) -> Tuple[str, bool]:
+    # Remove the doi: prefix if it exists
```

Comment:
nit: I don't think this comment adds any additional clarity to the code. I suggest removing.